examples/03_connectivity/plot_compare_decomposition.py

```diff
@@ -120,28 +115,21 @@
 
 ###############################################################################
 # Visualize the results
-# ----------------------
-from nilearn.plotting import find_xyz_cut_coords
-from nilearn.image import index_img
-
-names = {dict_learning: 'DictionaryLearning', canica: 'CanICA'}
-estimators = [canica, dict_learning]
-components_imgs = [canica_components_img, dictlearning_components_img]
-
-# Selecting specific maps to display: maps were manually chosen to be similar
-indices = {dict_learning: 8, canica: 14}
-# We select relevant cut coordinates for displaying
-cut_component = index_img(components_imgs[0], indices[dict_learning])
-cut_coords = find_xyz_cut_coords(cut_component)
-for estimator, components in zip(estimators, components_imgs):
-    # 4D plotting
-    plot_prob_atlas(components, view_type="filled_contours",
-                    title="%s" % names[estimator],
-                    cut_coords=cut_coords, colorbar=False)
-    # 3D plotting
-    plot_stat_map(index_img(components, indices[estimator]),
-                  title="%s" % names[estimator],
-                  cut_coords=cut_coords, colorbar=False)
+#
```

Comment:
If we remove this part, it's going to break the resting_state_networks.rst in the User Guide. It's fine if you want to remove it but we'll also need to edit that doc as well. (As of now, it's linking to the images created in the lines you removed).